contrib/local-test/Makefile

```diff
@@ -1,12 +1,17 @@
 DGRAPH_VERSION ?= local
 
 current_dir = $(shell pwd)
-
+local-dgraph-data: 
+	mkdir -p ~/local-dgraph-data
 up: ## Start the zero and alpha containers
 	DGRAPH_VERSION=$(DGRAPH_VERSION) docker-compose up -d && docker-compose logs --follow alpha zero keepalive
 
-up-with-lambda: ## Start the zero and alpha containers and a lambda container
+up-with-lambda: local-dgraph-data## Start the zero and alpha containers and a lambda container
 	DGRAPH_VERSION=$(DGRAPH_VERSION) docker-compose -f docker-compose.yml -f docker-compose-lambda.yml up -d && docker-compose logs --follow alpha zero lambda keepalive
+up-with-lambda-acl: local-dgraph-data ## Start the zero and alpha containers and a lambda container
+	mkdir -p ~/local-dgraph-data/acl
+	test -f ~/local-dgraph-data/acl/hmac_secret_file || openssl rand -hex 16 > ~/local-dgraph-data/acl/hmac_secret_file
```

Comment:
main comment: I think it would be cleaner to avoid this kind of magic and simply check in a `hmac_secret_file` into this directory.  Then we can modify your `docker-compose-acl.yml` as follows:

```
  alpha:
    volumes:
      - ~/local-dgraph-data:/dgraph
      - type: bind
        source: ./hmac_secret_file
        target: /dgraph/acl/hmac_secret_file
        read_only: true

    environment:
      DGRAPH_ALPHA_ACL: secret-file=/dgraph/acl/hmac_secret_file
```

This is so we don't have to generate a new secret every time, and we don't have to worry about whether a directory exists on the local system or not.